tests/acceptance/sunday-redirect-test.js

```diff
@@ -0,0 +1,21 @@
+import Ember from 'ember';
+import startApp from '../helpers/start-app';
+
+var application;
+
+module('Acceptance: SundayRedirect', {
+  setup: function() {
+    application = startApp();
+  },
+  teardown: function() {
+    Ember.run(application, 'destroy');
+  }
+});
+
+test('app/index.js should redirect root path to sunday', function() {
```

Comment:
We generally do not put "should" in test descriptions just for the sake of brevity. You can rename this test to "It redirects root path to /sunday".